eve/workers/compile_centos_8/Dockerfile

```diff
@@ -21,13 +25,13 @@ RUN yum install -y \
         zlib-devel
 
 # Installation of golang
-ARG GO_VERSION=1.14.2
+ARG GO_VERSION=1.14
```

Comment:
revert ?

Reply:
I used the same version as mentioned in the Dockerfile for centos8 in the repository scality/hdcontroller.

It got updated in between this PR and today. 

I suppose we should use the exact same version of goland in both repositories.